packages/terra-form-radio/tests/jest/RadioField.test.jsx

```diff
@@ -1,70 +1,163 @@
 import React from 'react';
 import ThemeContextProvider from 'terra-theme-context/lib/ThemeContextProvider';
+import uniqueid from 'lodash.uniqueid';
 
 /* eslint-disable-next-line import/no-extraneous-dependencies */
 import { shallowWithIntl, mountWithIntl } from 'terra-enzyme-intl';
 import RadioField from '../../src/RadioField';
 import Radio from '../../src/Radio';
 
+jest.mock('lodash.uniqueid');
 window.matchMedia = () => ({ matches: true });
 
+let userAgentGetter;
+beforeEach(() => {
+  userAgentGetter = jest.spyOn(window.navigator, 'userAgent', 'get');
+  uniqueid.mockImplementation(() => 'uuid123');
+});
+
 it('should render a default radio field', () => {
-  const radioField = (<RadioField legend="Default RadioField" />);
+  const radioField = <RadioField legend="Default RadioField" />;
   const wrapper = shallowWithIntl(radioField);
   expect(wrapper).toMatchSnapshot();
 });
 
+it('should render radio field with div element for Safari browser or Edg browser', () => {
+  userAgentGetter.mockReturnValue('Safari Edg');
+  const wrapper = shallowWithIntl(
+    <RadioField legend="Custom Message RadioField" />,
+  );
+  expect(wrapper.dive()).toMatchInlineSnapshot(`
+    <fieldset
+      className="radio-field"
+      id="terra-radio-group-uuid123"
+      required={false}
+    >
+      <div
+        className="legend-group"
+        id="terra-radio-field-description-uuid123"
+      >
+        <div
+          className="legend"
+        >
+          Custom Message RadioField
+          <span
+            className="error-icon-hidden"
+          />
+        </div>
+      </div>
+    </fieldset>
+  `);
+});
+
+it('should render radio field with legend element for Chrome browser', () => {
+  userAgentGetter.mockReturnValue('Chrome');
+  const wrapper = shallowWithIntl(
+    <RadioField legend="Custom Message RadioField" />,
+  );
```

Comment:
Nit: this can be on a single line.
```suggestion
  const wrapper = shallowWithIntl(<RadioField legend="Custom Message RadioField" />);
```

Reply:
i have made changes as per suggestions